spec/draper/configuration_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+module Draper
+  RSpec.describe Configuration do
+    it 'yields Draper on configure' do
+      Draper.configure { |config| expect(config).to be Draper }
+    end
+
+    it 'defaults default_controller to ApplicationController' do
+      skip
```

Comment:
These tests all pass every time when I run just this file. However, these last two tests cause the entire suite to fail with an error about leaking a double. Due to it being a state leaking issue, it's not a consistent test that fails.  The two different errors that keep seeing are:
![image](https://cloud.githubusercontent.com/assets/651946/24485397/cb97c09c-14c9-11e7-8cbd-b5dfba721837.png)

and 

![image](https://cloud.githubusercontent.com/assets/651946/24485400/d2505872-14c9-11e7-9591-cfaea78141b6.png)

I'm not sure how to go about fixing those as I'm not super familiar with the test setup on this project. I will continue to look into it, but could use some advice if someone has some.